ddtrace/contrib/botocore/patch.py

```diff
@@ -72,6 +61,8 @@
         "operations": collections.defaultdict(Config._HTTPServerConfig),
         "tag_no_params": asbool(os.getenv("DD_AWS_TAG_NO_PARAMS", default=False)),
         "instrument_internals": asbool(os.getenv("DD_BOTOCORE_INSTRUMENT_INTERNALS", default=False)),
+        "propagation_enabled": asbool(os.getenv("DD_BOTOCORE_PROPAGATION_ENABLED", default=False)),
```

Comment:
Can I ask why the default is False?

Reply:
Also I wonder if we want to keep using keyword `BOTOCORE`. Does `AWS` or `AWSSDK` makes more sense when we want to keep it consistent for tracers in other languages?

Reply:
The default is set to `False` since we cannot change the default user experience for the integration. Adding propagation by default would be considered a breaking change. As for the naming, I don't really have a preference but wanted to stay consistent. We have other configurations using `botocore` and we generally use the contrib name within the configuration name.

Reply:
I think our intentions are the same, that is, not to make it a breaking change. I'll need some time to test this for serverless (AWS Lambda) cases.